### DISCUSSION TITLE
M3 music audio quality

### DISCUSSION THREAD

**Discussion by e-Feitosa** [sircharlo/meeting-media-manager#1599] [Q&A] [answered]
🎶 In the neighboring congregation, the AV team said that there was a problem after they implemented the M3, but it is only when they are going to reproduce the chants, in the zoom, the music is oscillating. The question [doubt] is, could it be a problem related to the M3 or more likely is that problem in the zoom and to solve it you need to activate the high fidelity audio (music)? _If anyone had this problem and solved it, please comment here. Remembering that the zoom does not allow to set this fixed option, every time you need to click to activate the original sound._

**Zoom option active**
![image](https://user-images.githubusercontent.com/90984283/221945512-33bc1c77-a951-49f5-aafc-69c9503b4df2.png)

**Every time you open the zoom activate that original sound option**
![image](https://user-images.githubusercontent.com/90984283/221945245-207c39ca-b39c-49bd-8342-fee2196695b4.png)
⬇️
![image](https://user-images.githubusercontent.com/90984283/221945861-340c33db-84da-469a-96e7-199895a71a18.png)

**Comment by sircharlo** [comment, marked as answer]
Yup. Original sound needs to be enabled at all times for Zoom audio to be clear when playing music or videos.

**Reply by e-Feitosa** [reply]
@sircharlo, What draws attention is that this problem only happens with the songs (“Sing Out Joyfully”), the videos have normal sound, and a curious fact about this is that some have started to have problems in the last few weeks, before it was working without activating this commented feature above.

**Reply by mtdvlpr** [reply]
@e-Feitosa I suspect this has to do with a Zoom update and not so much M3.

**Reply by e-Feitosa** [reply]
@mtdvlpr, I also believe so. I just mentioned it here in case someone has the same problem and comes looking for it.

**Reply by alanfermtz** [reply]
To not playing around with original sound, just use the "Low" setting. This is entirely problem by zoom. We noticed this before even using m3.
